ml-agents/mlagents/trainers/ppo/policy.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, seed, brain, trainer_params, is_training, load):
                 m_size=self.m_size,
                 seed=seed,
                 stream_names=list(reward_signal_configs.keys()),
+                vis_encode_type=trainer_params["vis_encode_type"],
```

Comment:
Convert to EncoderType enum here and pass that through instead of the string type. Handle a missing value with .get(). For example

``` python
vis_encode_type = EncoderType(trainer_params.get("vis_encode_type", "default"))
```